code/game/jobs/job/job.dm

```diff
@@ -54,6 +54,7 @@
 	var/give_psionic_implant_on_join = TRUE // If psionic, will be implanted for control.
 
 	var/use_species_whitelist // If set, restricts the job to players with the given species whitelist. This does NOT restrict characters joining as the job to the species itself.
+	var/require_whitelist // If set, requires a separate whitelist to use the job.
```

Comment:
Might be better to change the variable name or specify in the comment that this is meant to be the string to check and not just a boolean "is this whitelisted or not" value. The latter is what I initially assumed before looking at the implementation in the AI job.

Since this is just semantics and I can't find anything else, I'll approve it anyway.

Reply:
Oh ye hang on